VideoWeb/VideoWeb.EventHub/Handlers/DisconnectedEventHandler.cs

```diff
@@ -19,7 +19,8 @@ public DisconnectedEventHandler(IHubContext<Hub.EventHub, IEventHubClient> hubCo
 
         protected override async Task PublishStatusAsync(CallbackEvent callbackEvent)
         {
-            await PublishParticipantDisconnectMessage().ConfigureAwait(false);
+            // skip disonnect events since judge available/unavailable statuses are tracked by other events
```

Comment:
Hi Shaed, Can we look at why the original issue is happening first before disabling this?